libs/core/src/lib/time-picker/time-picker.component.ts

```diff
@@ -220,6 +226,30 @@ export class TimePickerComponent<D>
     @Input()
     parseFormat: unknown;
 
+    /**
+     * Value state "success" aria message.
+     */
+    @Input()
+    valueStateSuccessMessage: string = this._valueStateAriaMessagesService.success;
```

Comment:
As `_valueStateAriaMessagesService` is Optional so please use `?.` for optional chaining here and below.

Reply:
ValueStateAriaMessageService is not optional
The service itself uses `provideIn: root`